docs/tools/cli/service/privatelink.rst

```diff
@@ -0,0 +1,423 @@
+    ==========================  =======================================================  =========================================  ======
+    vpce-svc-1234567890abc1234  com.amazonaws.vpce.us-east-1.vpce-svc-1234567890abc1234  arn:aws:iam::123456789012:user/cloud_user  active
+
+.. _avn_service_privatelink_azure_connection_approve:
+
+``avn service privatelink azure connection approve``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Approves pending PrivateLink connection endpoint.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``privatelink_connection_id``
+    - The Aiven PrivateLink connection ID
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--format``
+    - Format of the output string
+
+**Example:** Approve PrivateLink ``plc12345abcdef`` for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink azure connection approve kafka-12a3b4c5 plc12345abcdef
+
+An example of output:
+
+.. code:: text
+
+    PRIVATE_ENDPOINT_ID                                                                                                                       PRIVATELINK_CONNECTION_ID  STATE          USER_IP_ADDRESS
+    ========================================================================================================================================  =========================  =============  ===============
+    /subscriptions/12345678-90ab-cdef-0987-6543210abcde/resourceGroups/group-eastus/providers/Microsoft.Network/privateEndpoints/pl-endpoint  plc12345abcdef             user-approved  null
+
+.. _avn_service_privatelink_azure_connection_list:
+
+``avn service privatelink azure connection list``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink connection for a service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--format``
+    - Format of the output string
+
+**Example:** List PrivateLink for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink azure connection list kafka-12a3b4c5
+
+An example of output:
+
+.. code:: text
+
+    PRIVATELINK_CONNECTION_ID  PRIVATE_ENDPOINT_ID                                                                                                                       STATE                  USER_IP_ADDRESS
+    =========================  ========================================================================================================================================  =====================  ===============
+    plc12345abcdef             /subscriptions/12345678-90ab-cdef-0987-6543210abcde/resourceGroups/group-eastus/providers/Microsoft.Network/privateEndpoints/pl-endpoint  pending-user-approval  null
```

Comment:
Question: should we shorten the `/subscription` to make it better readable?

Reply:
I can certainly do it, but it will lose the original format and length. Let me know if you want me to do it.

Reply:
I prefer clarity, please leave it like that!